### DISCUSSION TITLE
Power lost option

### DISCUSSION THREAD

**Discussion by ims21** [Jyers/Marlin#1662] [Q&A]
Has this option sence for printers with added 'USB' capacitors only or is it Ender 3v2 power source able moving Z up, when is power interrupted ?

**Comment by dante-35** [comment]
This function creates a file in the sd card and saves its curren position whenever z changes 0.05mm.
When power outage resolves it reads the file in the sd card and continues processing gcode from last saved position.
Answer to your question is printer only rely on its own power supply.
But thre are tutorials on you tube how to add some capacitor circuits and notifiying pirinter when power outage occures.
That way before shutting down printer parks its head an saves current location to sc card.
